<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="ANKUSH-PC" failures="1" tests="3" name="TestNGTest" time="5.073" errors="0" timestamp="27 Oct 2018 08:01:27 GMT">
  <testcase classname="com.testng.TestNG1" name="TestNG1m2" time="2.016"/>
  <testcase classname="com.testng.TestNG1" name="TestNG1m4" time="2.013"/>
  <testcase classname="com.testng.TestNG1" name="TestNG1m1" time="1.019">
    <failure type="org.testng.internal.thread.ThreadTimeoutException" message="Method com.testng.TestNG1.TestNG1m1() didn&amp;apos;t finish within the time-out 1000">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method com.testng.TestNG1.TestNG1m1() didn't finish within the time-out 1000
at java.base@10.0.2/java.lang.Thread.interrupt(Unknown Source)
at java.base@10.0.2/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.base@10.0.2/java.util.concurrent.FutureTask.run(Unknown Source)
at java.base@10.0.2/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.base@10.0.2/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.base@10.0.2/java.lang.Thread.run(Unknown Source)
... Removed 2 stack frames]]>
    </failure>
  </testcase> <!-- TestNG1m1 -->
  <testcase classname="com.testng.TestNG1" name="TestNG1m3" time="0.0">
    <ignored/>
  </testcase> <!-- TestNG1m3 -->
</testsuite> <!-- TestNGTest -->
